code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -419,31 +419,29 @@
 	name = "Stable Mutation Toxin"
 	description = "A humanizing toxin."
 	color = "#5EFF3B" //RGB: 94, 255, 59
-	metabolization_rate = INFINITY //So it instantly removes all of itself
+	metabolization_rate = 0 //Doesn't metabolize. Gets destroyed on a timer.
```

Comment:
where does it get deleted?

Reply:
Good, point... could have sworn I did qdel(src) at the end of mutate but apparently didn't.

Reply:
Done